src/scicookie/cookiecutter.json

```diff
@@ -19,6 +19,14 @@
   "command_line_interface": ["None", "Click", "Argparse"],
   "documentation_engine": ["mkdocs", "sphinx", "jupyter-book", "quarto"],
   "documentation_url": "{{ cookiecutter.project_url }}",
+  "mkdocs_theme": ["Material", "Cinder", "Mkdocs", "Readthedocs"],
```

Comment:
I think that the first option should be "Default" for all the four variables about themes (mkdocs, sphinx and jupyter book)